packages/nimble-blazor/README.md

```diff
@@ -2,9 +2,10 @@
     <p><b>ni | nimble | blazor</b></p>
 </div>
 
-# Nimble Blazor
+# Nimble and Spright Blazor
```

Comment:
In a follow-up PR lets rename the package and sln and split this into two READMEs in the right spots